docs-gb/kubernetes/resources/seldonconfig.md

```diff
@@ -28,4 +77,282 @@ The default configuration is shown below.
 
 ```yaml
 # operator/config/seldonconfigs/default.yaml
+apiVersion: mlops.seldon.io/v1alpha1
```

Comment:
Please reference the file via a @github-files/github-code-block. As much as possible, we want to change things in a single place, with the files in the repo being the source-of truth.

Reply:
unable to find the file

Reply:
Here: https://github.com/SeldonIO/seldon-core/blob/v2/operator/config/serverconfigs/mlserver.yaml (see path commented at the top). Perhaps worth considering if all source files need to be within the docs folder?